project-base/app/src/DataFixtures/Demo/SettingValueDataFixture.php

```diff
@@ -127,7 +127,7 @@ public function load(ObjectManager $manager)
             );
             $this->setting->setForDomain(
                 SeoSettingFacade::SEO_ROBOTS_TXT_CONTENT,
-                'Disallow: /admin',
+                'Disallow: *?filter=',
```

Comment:
This way admin can now be revealed to the public right? Would not be better to use something like:
/admin*

Reply:
actually, no... robots.txt is merely a recommendation. 
so having the URL of the administration here actually exposes the URL to the world (to anyone, who opens this file). 
From this point of view, I don't think the URL of the administration must not be in the robots.txt file at all. 

Besides, the /admin route itself already contains the proper `robots` meta tag (`<meta name="robots" content="noindex, nofollow">`)

_I think robots.txt should be renamed to hacker_must_visit.txt_